src/dev/amdgpu/amdgpu_defines.hh

```diff
@@ -61,36 +71,11 @@ constexpr int MMIO_BAR = 5;
 constexpr uint32_t VGA_ROM_DEFAULT = 0xc0000;
 constexpr uint32_t ROM_SIZE = 0x20000;        // 128kB
 
-/* SDMA base, size, mmio offset shift. */
```

Comment:
I'm assuming these are all just removed in ROCm 6?  If so, does removing them break GPUSE?